examples/pycaffe/caffenet.py

```diff
@@ -45,10 +46,10 @@ def caffenet(lmdb, batch_size=256, include_acc=False):
 
 def make_net():
     with open('train.prototxt', 'w') as f:
-        print >>f, caffenet('/path/to/caffe-train-lmdb')
+        print( caffenet('/path/to/caffe-train-lmdb'), file=f )
```

Comment:
Please follow PEP8 here and below: http://legacy.python.org/dev/peps/pep-0008/#pet-peeves.